components/sidenav/index.css

```diff
@@ -92,7 +92,6 @@
 		--spectrum-sidenav-top-level-line-height: var(--spectrum-cjk-line-height-100);
 	}
 
-	--spectrum-sidenav-header-font-family: var(--spectrum-sans-font-family-stack);
```

Comment:
Oh, I wonder if we should be using this variable and aren't. This might be one where we want to add the font-family property rather than remove the variable.

Reply:
Good catch! I applied it alongside the other tokens in that block — that class was missing a `font-family` declaration.